modules/cli/pages/cbstats/cbstats-collections-details.adoc

```diff
@@ -67,15 +104,23 @@ Each `scopeID` or `collectionID` is a hexadecimal number, prefixed by `0x`.
 Each `datumLabel` is a string.
 Each data key is concluded with a colon, and the associated value is presented to the right of the colon.
 
-The initial two lines provide information that identifies existing collections.
-Two are listed, which are the `_default` collection, and `MyCollection`.
-Each is identified with hexadecimal numbers: the `_default` collection is identified as `0x0` (residing within scope `0x0`); and `MyCollection` as `0x9` (residing within scope `0x8`).
+The initial lines provide information that identifies existing collections in the vBucket.
+Each is identified with hexadecimal numbers: for example, the `_default` collection is identified as `0x0` (residing within scope `0x0`); and `MyCollection` as `0x9` (residing within scope `0x8`).
```

Comment:
This no longer matches the new output, MyCollection is now collection 0x17 in scope 0x0.

Reply:
Thanks!